src/server/api/routers/auth.ts

```
@@ -1,30 +1,34 @@
 import { TRPCError } from "@trpc/server";
+import { SignJWT } from "jose";
 import { z } from "zod";
 
 import { UserVerificationErrors } from "~/_types/errors";
-import { logger } from "~/_utils";
+import { instagramHandleOptional, logger, phoneSchema, socialHandleOptional } from "~/_utils";
 import { env } from "~/env";
 import { createUser, createVerificationToken, verifyUserEmail } from "~/server/_db";
+import { generateVerificationCode } from "~/server/_utils";
+import { captureEvent } from "~/server/_lib/posthog";
```

ESLint

[error] `~/server/_lib/posthog` import should occur before import of `~/server/_utils`

(import/order)